app/controllers/concerns/staff_response_context.rb

```diff
@@ -3,8 +3,17 @@ module StaffResponseContext
 
 private
 
-  def load_visit
-    @visit ||= scoped_visit
+  def memoised_visit
+    @_visit ||= scoped_visit
+  end
+
+  def decorate_visit(visit)
+    @_decorated_visit ||=
```

Comment:
No, because is not always the memoised visit. We use a visit with eager loaded associations in `prision/visits#show` action